ethereum/blockcreation/src/test/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockTransactionSelectorTest.java

```diff
@@ -737,6 +737,37 @@ public void transactionWithIncorrectNonceRemainsInPoolAndNotSelected() {
                     TransactionInvalidReason.NONCE_TOO_HIGH.name())));
   }
 
+  @Test
+  public void shouldNotSelectTransactionsWithPriorityFeeLessThanConfig() {
+    ProcessableBlockHeader blockHeader = createBlock(5_000_000, Wei.ONE);
+    miningParameters.setMinPriorityFeePerGas(Wei.of(7));
+    final Transaction tx1 = createTransaction(1, Wei.of(8), 100_000);
+    ensureTransactionIsValid(tx1);
+    final Transaction tx2 = createTransaction(2, Wei.of(9), 100_000);
+    ensureTransactionIsValid(tx2);
+    // transaction tx3 should not be selected
+    final Transaction tx3 = createTransaction(3, Wei.of(7), 100_000);
+    ensureTransactionIsValid(tx3);
+    transactionPool.addRemoteTransactions(List.of(tx1, tx2, tx3));
```

Comment:
Could you add another test with EIP1559 txs?

Reply:
done